src/hermes/commands/init/base.py

```diff
@@ -0,0 +1,246 @@
+            return
+
+        sc.echo(f"Starting to initialize HERMES in {self.folder_info.absolute_path}")
+
+        # Abort if there is no git
+        if not self.folder_info.has_git:
+            sc.echo("The current directory already has no `.git` subdirectory. "
+                       "Please execute `hermes init` in the root directory of your git project.")
+            return
+
+        # Abort if neither GitHub nor gitlab is used
+        if not (self.folder_info.uses_github or self.folder_info.uses_gitlab):
+            sc.echo("Your git project ({}) is not connected to github or gitlab. It is mandatory for HERMES to "
+                       "use one of those hosting services.".format(self.folder_info.git_remote_url))
+            return
+
+        # Abort if there is already a hermes.toml
+        if self.folder_info.has_hermes_toml:
+            sc.echo("The current directory already has a `hermes.toml`. "
+                    "It seems like HERMES was already initialized for this project.")
+            if not sc.confirm("Do you want to initialize Hermes anyway? (files will be overwritten)"):
+                return
+
+        # Creating the citation File
+        if not self.folder_info.has_citation_cff:
+            citation_cff_url = "https://citation-file-format.github.io/cff-initializer-javascript/#/"
+            sc.echo("Your project does not contain a `CITATION.cff` file (yet). It would be very helpful for "
+                       "saving important metadata which is necessary for publishing.")
+            create_cff_now = sc.confirm("Do you want to create a `CITATION.cff` file now?", default=True)
+            if create_cff_now:
+                sc.echo("{} to create the file. Then move it into the project folder before you continue.".format(
+                    create_console_hyperlink(citation_cff_url, "Click here")))
+                done_creating_cff = sc.confirm("Are you done?", default=False)
+                while not done_creating_cff:
+                    done_creating_cff = sc.confirm("Are you done now?", default=True)
+                self.refresh_folder_info()
+                if self.folder_info.has_citation_cff:
+                    sc.echo("Good job!")
+                else:
+                    sc.echo("Hey you lied to me :( Don't forget to add the `CITATION.cff` file later!")
+            else:
+                sc.echo("You better do it later or HERMES won't work properly.")
+                sc.echo("You can {} to create the file. Then move it into the project folder.".format(
+                    create_console_hyperlink(citation_cff_url, "click here")))
+        else:
+            sc.echo("Your project already contains a `CITATION.cff` file. Nice!")
+
+        # Creating the hermes.toml file
+        hermes_toml_raw_url = "https://raw.githubusercontent.com/nheeb/zenodo-test/main/hermes.toml"
+        download_file_from_url(hermes_toml_raw_url, os.path.join(os.getcwd(), "hermes.toml"))
```

Comment:
This file should be part of this repo and then loaded via package data. There is no need to download it.

Reply:
Should it even be generated / serialized from a config object? (I agree we shouldn't download it)